packages/connectivity/example/android/settings.gradle

```diff
@@ -10,6 +10,8 @@ if (pluginsFile.exists()) {
 
 plugins.each { name, path ->
     def pluginDirectory = flutterProjectRoot.resolve(path).resolve('android').toFile()
-    include ":$name"
-    project(":$name").projectDir = pluginDirectory
+    if (pluginDirectory.exists()) {
+        include ":$name"
+        project(":$name").projectDir = pluginDirectory
+    }
```

Comment:
If this is something every single app including this plugin needs or else it will fail to compile, this probably shouldn't merge because it will break existing apps. Apps currently using `connectivity` don't have this block and won't know to update this when their build breaks.

Reply:
Yeah, I reverted this change and changed the connectivity_macos/pubspec.yaml to require flutter >= 1.13.0 where this is not an issue anymore. Will have to wait until https://github.com/flutter/plugins/pull/2334 is landed to land this as well.